### PR TITLE
HPCC-14389 ProcessPerSlave change and thor related configxml cleanup

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner michael.gardner@lexisnexis.com

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@jakesmith @garonsky Please review when you have time

File: initfiles/componentfiles/configxml/setvars_linux.xsl [hunk 1]
```diff
@@ -42,108 +35,66 @@
...
+                        <xsl:with-param name="attribute" select="Storage/@breakoutLimit"/>
+                        <xsl:with-param name="default" select="'3600'"/>
+                     </xsl:call-template>
+export refreshrate=<xsl:call-template name="setOrDefault">
+                        <xsl:with-param name="attribute" select="Storage/@refreshRate"/>
+                        <xsl:with-param name="default" select="'3'"/>
+                     </xsl:call-template>
 export autoSwapNode=<xsl:choose>
            <xsl:when test="SwapNode/@AutoSwapNode='true'">1</xsl:when>
            <xsl:when test="SwapNode/@AutoSwapNode='1'">1</xsl:when>
```
Comment (jakesmith) [inline initfiles/componentfiles/configxml/setvars_linux.xsl]: Not new, but don't see why this property in particular vs others is checking '1' ,

**Comment by jakesmith** [comment]
Looks a lot cleaner.
@garonsky , please double check

**Comment by Michael-Gardner** [comment]
@jakesmith I've gone ahead and modified the autoSwapNode code to do a boolean() check on the elements value, and then output true or false.  I've gone ahead and changed the appropriate code in init_thor to make this change work.  All the c++ code referencing AutoSwapNode is using getBoolean so we should be fine there.

**Comment by Michael-Gardner** [comment]
@garonsky can you please give the xslt a glance before I @ Richard?

**Comment by garonsky** [comment]
Looks good.

**Comment by Michael-Gardner** [comment]
@richardkchapman Ready for a merge :+1:

**Pull request merged**